lms/djangoapps/teams/static/teams/js/spec/topic_header_spec.js

```diff
@@ -0,0 +1,71 @@
+define(['teams/js/views/topic_header', 'teams/js/collections/topic'],
+    function (TopicHeader, TopicCollection) {
+        'use strict';
+        describe('TopicHeader', function () {
+            var topicHeader,
+                courseId = 'testX/test/test_20XX',
+                newCollection = function (size, perPage) {
+                    var pageSize = 5,
+                        results = _.map(_.range(size), function (i) {
+                            return {
+                                "description": "description " + i,
+                                "name": "topic " + i,
+                                "id": "id " + i
+                            };
+                        });
+                    var collection = new TopicCollection(
+                        {
+                            count: results.length,
+                            num_pages: results.length / pageSize,
+                            current_page: 1,
+                            start: 0,
+                            results: _.first(results, perPage)
+                        },
+                        {course_id: courseId, parse: true}
+                    );
+                    collection.start = 0;
+                    collection.totalCount = results.length;
+                    return collection;
+                };
+
+            it('can load templates', function () {
```

Comment:
Nit: rename to 'can render itself'; it's not just loading a template but also populating it with data and appending itself to the DOM.